ddl/backfilling.go

```diff
@@ -805,12 +806,21 @@ func (b *backfillScheduler) Close() {
 //
 // The above operations are completed in a transaction.
 // Finally, update the concurrent processing of the total number of rows, and store the completed handle value.
-func (dc *ddlCtx) writePhysicalTableRecord(sessPool *sessionPool, t table.PhysicalTable, bfWorkerType backfillWorkerType, reorgInfo *reorgInfo) error {
+func (dc *ddlCtx) writePhysicalTableRecord(sess *session, sessPool *sessionPool, t table.PhysicalTable, bfWorkerType backfillWorkerType, reorgInfo *reorgInfo) error {
 	job := reorgInfo.Job
 	totalAddedCount := job.GetRowCount()
 
 	startKey, endKey := reorgInfo.StartKey, reorgInfo.EndKey
 	sessCtx := newContext(reorgInfo.d.store)
+	m, err := sess.Txn(true)
+	if err != nil {
+		return err
+	}
+	dbInfo, err := meta.NewMeta(m).GetDatabase(job.SchemaID)
```

Comment:
Could we put it to `onCreateIndex` which we needn't do more than once? It can be passed through `reorgInfo`. And we needn't `GetDatabase` in this background.

Reply:
This PR also has other problems, let me fix it.

Reply:
I think it should be fixed by https://github.com/pingcap/tidb/issues/39826

Reply:
Updated, opened another issue to trace rename issue, https://github.com/pingcap/tidb/issues/39826. PTAL  @zimulala